src/main/java/org/embulk/filter/column/ColumnFilterPlugin.java

```diff
@@ -117,7 +120,7 @@ static Schema buildOutputSchema(PluginTask task, Schema inputSchema)
                 boolean matched = false;
                 for (ColumnConfig dropColumn : dropColumns) {
                     // skip json path notation to build outputSchema
-                    if (dropColumn.getName().startsWith("$.")) {
+                    if (PathCompiler.isDocContext(dropColumn.getName())) {
```

Comment:
> So, It looks better to try `PathCompiler.compile` and put paths if no error occurs, rather than just checking starting dollar.

* If path isn't starts with dollar, the original implementation adds the dollar (I don't know why it is needed). I would like to drop this feature. https://github.com/jayway/JsonPath/blob/master/json-path/src/main/java/com/jayway/jsonpath/internal/path/PathCompiler.java#L63
* Usually, I think path isn't jsonpath expression. I think it brings a little performance problem if it always throw the exception.
* In general, `is` prefix should be used for boolean variables and methods isn't it? (BTW, I should be changed type declaration to the boolean) https://wiki.eclipse.org/Recommenders/CodingConventions#is_prefix_should_be_used_for_boolean_variables_and_methods.